Podcast/DiscoverTopicViewController.swift

```diff
@@ -286,8 +286,7 @@ extension DiscoverTopicViewController: DiscoverTableViewHeaderDelegate {
         switch sender.tag {
         case seriesHeaderTag:
             guard let id = topic.id else { break }
-            let vc = BrowseSeriesViewController(mediaType: .topic(id: id))
-            vc.series = topSeries
+            let vc = BrowseSeriesViewController(mediaType: .topic(id: id), series: topSeries)
```

Comment:
it's fine for here but in the future let's try to make our local variables more descriptive (especially since we don't comment our code heavily)